src/EventStore.Core/Index/TableIndex.cs

```diff
@@ -466,7 +466,7 @@ public class TableIndex : ITableIndex {
 						_awaitingMemTables[j] = new TableItem(ptable,
 							tableItem.PrepareCheckpoint,
 							tableItem.CommitCheckpoint,
-							tableItem.Level);
+							tableItem.Level, false);
```

Comment:
@jageall Instead of passing `false` here, would it be better to pass in `tableItem.IsFromIndexMap`, then in the first for loop, skip those awaiting tables where `IsFromIndexMap` is `true`?

Reply:
I thought about that, but as the code cannot currently be used, I went with minimal changes. I'd suggest we follow up on this by either removing the method that cannot be called, or making this and some other changes when we test and make this option available to consumers.